docs/manifest.json

```diff
@@ -346,10 +346,18 @@
     { "title": "Next.js", "icon": "nextjs", "root": "references/nextjs" },
     [
       ["Overview", "/references/nextjs/overview"],
-      ["authMiddleware()", "/references/nextjs/auth-middleware"],
-      ["withClerkMiddleware()", "/references/nextjs/with-clerk-middleware"],
+      "# App Router",
+      ["auth()", "/references/nextjs/auth"],
+      ["currentUser()", "/references/nextjs/current-user"],
+      ["Route Handlers", "/references/nextjs/route-handlers"],
+      ["Server Actions", "/references/nextjs/server-actions"],
+      "# Pages Router",
       ["getAuth()", "/references/nextjs/get-auth"],
-      ["buildClerkProps()", "/references/nextjs/build-clerk-props"]
+      ["buildClerkProps()", "/references/nextjs/build-clerk-props"],
+      ["withClerkMiddleware()", "/references/nextjs/with-clerk-middleware"],
+      "---",
+      ["authMiddleware()", "/references/nextjs/auth-middleware"],
```

Comment:
Can we move this link above the `App Router` heading, and move `withClerkMiddleware()` below the separator. The `withClerkMiddleware()` method is deprecated, and `authMiddleware()` is used across both routers and is one of the first things that should be implemented.